### PR TITLE
Edited the story.txt and madlib.txt

### PR DESCRIPTION
**Pull request by AkashSeshadri**
Edited the story.txt and madlib.txt

### CODE REVIEW

**Comment by saranshagarwal202** [comment]
Looks good, will merge this.

File: madlibs_list.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
-1. Friend's Name
-2. Adjective
-3. Adjective
-4. Color
-5. Month
+1. Friend's Name - John
+2. Adjective - big
+3. Adjective - beautiful
+4. Color - red
+5. Month - june
```
Comment (dmaywald) [inline madlibs_list.txt:5]: Make sure that June is capitalized

File: story.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
 Title: The Galactic Adventure
 
-Dear [1],
+Dear John,
 
-I am writing to you from a [2] spaceship in a [3] galaxy. I found myself here one day after
-going for a ride on a [4] rocket in [5]. There are [6] [7] and [8] of [9] here! In the
+I am writing to you from a big spaceship in a beautiful galaxy. I found myself here one day after
+going for a ride on a red rocket in june. There are [6] [7] and [8] of [9] here! In the
```
Comment (dmaywald) [inline story.txt:6]: 'june' should be capitalized

**Review by dmaywald — COMMENTED** [review]
Excellent mad libs, just make sure 'june' is capitalized!

**Pull request merged**